cScripts/CavFnc/functions/modules/fn_moduleCallEndex.sqf

```diff
@@ -11,15 +13,73 @@
  *
  * Public: No
  */
+ 
+//Module Options
+private _dialogResults = [
+	"Endex Options",
+	[
+		["Set Player Weapons To Safe", ["Yes", "No"], 0],
+		["Pacify AI", ["Yes", "No"], 0],
+		["Hint Players That Fire", ["Yes", "No"], 0]
+	]
+] call Ares_fnc_showChooseDialog;
 
-"ENDEX ENDEX ENDEX" remoteExecCall ["systemChat", 0];
-"DEBRIEF IN GAME" remoteExecCall ["systemChat", 0];
-"LEADERS DEBRIEF IN CHANNEL BELOW" remoteExecCall ["systemChat", 0];
+if (count _dialogResults == 0) exitWith {};
 
+//handles results and turns them into booleans
+_dialogResults params ["_weaponsSafe", "_aiPacified", "_VCSTIME"];
+
+private _weaponsSafe = _weaponsSafe isEqualTo 0;
+private _aiPacified = _aiPacified isEqualTo 0;
+private _VCSTIME = _VCSTIME isEqualTo 0;
+
+
+//systemChat Endex message
+"ENDEX ENDEX ENDEX" remoteExecCall ["systemChat", 0]; //kept 0 for server logs
```

Comment:
chat messages is not log what i know of.
Add:
`"Mission have been endex" remoteExecCall [FUNC(logInfo), 2];` to execute it on only the server.
and use -2 to execute it to all clients and not on the server.
>When number is negative, the effect is inverted. -2 means execute on every client but not the server.